e2e-tests/demand-backup-physical/run

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+
+set -o errexit
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+
+run_backup() {
```

Comment:
Not sure why we had to include these 3 functions here if we have similar in `functions`?